safe_transaction_service/history/models.py

```diff
@@ -107,6 +117,10 @@ def parse(tx_type: str):
             raise ValueError(f"{tx_type} is not a valid InternalTxType")
 
 
+class IndexingStatusType(Enum):
+    ERC20_721_EVENTS = 0
```

Comment:
Whis is the purpose of this Enum with only one option? Thinking to include `masterCopy` indexing in this table also?

Reply:
Maybe in the future, yes 😉